Kitodo/src/main/java/de/sub/goobi/helper/BatchStepHelper.java

```diff
@@ -565,10 +567,11 @@ public String batchDurchBenutzerAbschliessen() throws DAOException, DataExceptio
                     MetadatenImagesHelper mih = new MetadatenImagesHelper(null, null);
                     try {
                         if (!mih.checkIfImagesValid(s.getProcess().getTitle(),
-                                serviceManager.getProcessService().getImagesOrigDirectory(false, s.getProcess()))) {
+                            serviceManager.getProcessService().getImagesOrigDirectory(false, s.getProcess()))) {
                             error = true;
                         }
                     } catch (Exception e) {
+                        logger.error(e.getMessage(), e);
```

Comment:
Please replace this log call and following `Helper.setFehlermeldung` with `Helper.setErrorMessage` which bundles both calls.